src/components/Form.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+
+const Form = (props) => {
+
+    const [selectedGenre, setSelectedGenre] = useState("placeholder");
+
+const handleChange = (e) =>{
+    
+    setSelectedGenre(e.target.value);
+    
+}
+return (
+<>
+ <form onSubmit={ (e) => { props.handleSubmit (e, selectedGenre) } }>
```

Comment:
consider indenting the child elements to their parent elements - this could also be a formatting thing on github